packages/web-runtime/src/components/SidebarNav/SidebarNav.vue

```diff
@@ -149,7 +147,7 @@ export default {
   max-width: 230px !important;
 }
 .oc-app-navigation-collapsed {
-  min-width: 58px !important;
-  max-width: 58px !important;
+  min-width: 62px !important;
```

Comment:
This fixes the uneven padding which is mentioned here: https://github.com/owncloud/web/issues/7525